pulpcore/cli/deb/distribution.py

```diff
@@ -0,0 +1,74 @@
+import gettext
+
+import click
+from pulpcore.cli.common.context import PulpContext, pass_pulp_context
+from pulpcore.cli.common.generic import (
+    base_path_contains_option,
+    base_path_option,
+    create_command,
+    destroy_command,
+    href_option,
+    label_command,
+    label_select_option,
+    list_command,
+    name_option,
+    resource_option,
+    show_command,
+    update_command,
+)
+
+from pulpcore.cli.deb.context import PulpAptDistributionContext, PulpAptRepositoryContext
+
+_ = gettext.gettext
+
+
+repository_option = resource_option(
+    "--repository",
+    default_plugin="deb",
+    default_type="apt",
+    context_table={"deb:apt": PulpAptRepositoryContext},
+    help=_(
+        "Repository to be used for auto-publishing."
```

Comment:
Like I said, `pulp_deb` does not currently support auto-publishing. Does it make sense to add this?

Reply:
This is a wrong help text. It's the repository for auto-distributing, which every publication based plugin supports.

Reply:
I always need to pause and think, to remember which of those terms is which...